LocalSettings.php

```diff
@@ -566,19 +590,16 @@
 	],
 	'wgCompressRevisions' => [
 		'default' => false,
-		'absurdopediawiki' => true,
 		'allthetropeswiki' => true,
 		'altversewiki' => true,
 		'americangirldollswiki' => true,
-		'animatedfeetwiki' => true,
 		'animebathswiki' => true,
-		'baobabarchiveswiki' => true,
 		'beidipediawiki' => true,
 		'buswiki' => true,
 		'commonwealthwiki' => true,
 		'crappygameswiki' => true,
-		'crystalmaidenswiki' => true,
 		'cwarswiki' => true,
+		'drawnfeetwiki' => true,
```

Comment:
Why is this being set? I don't see it being reordered.

Reply:
Because there was a previous wiki which was removed, because it redirected to this one.

Reply:
oh I see.